src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1624,6 +1624,7 @@ void goto_instrument_parse_optionst::help()
     "Safety checks:\n"
     " --no-assertions              ignore user assertions\n"
     HELP_GOTO_CHECK
+    " --enum-range-check           checks that all enum type expressions have values in the enum range\n" // NOLINT(*)
```

Comment:
Please add this to `HELP_GOTO_CHECK` rather than locally adding it here.

Reply:
Thanks